gcp-observability/src/main/java/io/grpc/gcp/observability/GcpObservability.java

```diff
@@ -73,17 +116,104 @@ public void close() {
       if (instance == null) {
         throw new IllegalStateException("GcpObservability already closed!");
       }
+      unRegisterStackDriverExporter();
```

Comment:
After this can one call `registerStackDriverExporter` again? So e.g. can one call grpcInit then close and then grpcInit again?

Reply:
Wrt `StackdriverExporter` we will be to call again after `unregistering`, yet to be verified with monitored resource detection.